src/features/user/Widget/DonationLink/CustomSnackBar.tsx

```diff
@@ -0,0 +1,47 @@
+import Snackbar from '@mui/material/Snackbar';
+import MuiAlert from '@mui/material/Alert';
+import i18next from '../../../../../i18n';
+import { ReactElement } from 'react';
+import styled from '@emotion/styled';
+
+interface SnackBarProps {
+  SnackBartext: String;
+  isVisible: boolean;
+  handleClose: () => void;
+}
+const { useTranslation } = i18next;
+
+const Alert = styled(MuiAlert)(({ theme }) => {
+  return {
+    backgroundColor: theme.palette.primary.main,
+  };
+});
+
+export default function CustomizedSnackbars({
```

Comment:
Match the default export name to the filename - it makes it easier to debug the component tree.